Source/Internal/API/APIEndpoints.swift

```diff
@@ -1,7 +1,7 @@
 //
 //  Endpoints.swift
 //
-//  Copyright (c) 2018 Virtusize AB
+//  Copyright (c) 2018-20 Virtusize KK
```

Comment:
just a ninja comment.
Facebook used to follow the `2018-present` pattern not to have to update everything every year BUT now they drop years entirely. Here is the PR (https://github.com/facebook/react/pull/13593) but unfortunately it just says 
```
Our newest copyright header guidelines advise we drop the year from the notice.
```
Maybe we can drop years as well? cc: @zr0z

Reply:
That's a good question, from a licensing point of view here, we don't necessarily need the year.

At the very least xxxx-present makes a lot of sense.

Reply:
The good thing about xxxx-present is that we don't need to change it for future years. So let's change it to 2018-present, shall we?